JS/article.js

```diff
@@ -1,24 +1,156 @@
-let res_nav = document.getElementById("responsive_nav");
-let hum = document.getElementById("fa-bars");
+let article_id = new URLSearchParams(window.location.search).get("id")
+let user = localStorage.getItem('user')
+let user_parsed = JSON.parse(user)
+
+let article_title = document.getElementById('article_title')
+let small_description = document.getElementById('small_description')
+let author_name = document.getElementById('author_name')
+let author_image = document.getElementById('author_image')
+let publish_date = document.getElementById('publish_date')
+let cover_photo = document.getElementById('cover_photo')
+let article_body = document.getElementById('article_body')
+let likes = document.getElementById('no_likes')
+let comments = document.getElementById('no_comments')
+let comments_list = document.getElementById('comments_list')
+let liking_icon = document.getElementById("liking_icon")
+let like_icon = document.getElementById('like_icon')
+let like_button = document.getElementById('like_button')
+let article;
```

Comment:
Instead of using snake_case try to use camelCase for the best practice in javascript.